README.md

```diff
@@ -39,7 +39,7 @@ Options:
 #### Run NW.js
 During development you can run NW.js with `nwbuild -r path/to/your/younwapp/`
 
-Or use the module:
+## Module Usage
```

Comment:
Maybe, but this way it's a noun both times before "usage"; spaghetti/linguine imo